spec.md

```diff
@@ -513,12 +513,12 @@ function fooBarBaz($arg1, &$arg2, $arg3 = [])
 }
 ```
 
-### 4.5 Method and Function Arguments
+### 4.5 Method and Function Parameters
```

Comment:
Here describe arguments and parameters both. In PHP used caption [Function arguments](https://www.php.net/manual/en/functions.arguments.php). May be in PER save caption "Method and Function Arguments" also?